PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/common/SqlServerNameValidator.java

```diff
@@ -4,24 +4,27 @@
  */
 package com.microsoft.azure.toolkit.intellij.sqlserver.common;
 
+import com.azure.core.management.exception.ManagementException;
 import com.microsoft.azure.toolkit.intellij.database.ServerNameTextField;
 import com.microsoft.azure.toolkit.lib.Azure;
 import com.microsoft.azure.toolkit.lib.common.entity.CheckNameAvailabilityResultEntity;
 import com.microsoft.azure.toolkit.lib.common.form.AzureValidationInfo;
 import com.microsoft.azure.toolkit.lib.sqlserver.service.AzureSqlServer;
 
 import java.util.function.Function;
-import java.util.regex.Pattern;
 
 public class SqlServerNameValidator implements Function<ServerNameTextField, AzureValidationInfo> {
 
-    private static final Pattern PATTERN = Pattern.compile("^[a-z0-9][a-z0-9-]+[a-z0-9]$");
-
     @Override
     public AzureValidationInfo apply(ServerNameTextField textField) {
         final String value = textField.getValue();
         // validate availability
-        CheckNameAvailabilityResultEntity resultEntity = Azure.az(AzureSqlServer.class).checkNameAvailability(textField.getSubscriptionId(), value);
+        CheckNameAvailabilityResultEntity resultEntity = null;
+        try {
+            resultEntity = Azure.az(AzureSqlServer.class).checkNameAvailability(textField.getSubscriptionId(), value);
+        } catch (ManagementException e) {
+            return AzureValidationInfo.builder().input(textField).message(e.getMessage()).type(AzureValidationInfo.Type.ERROR).build();
```

Comment:
merge the error handling of CloudException and ManagementException, they are same

Reply:
It will only exist one of them. for mysql (track1) --> CloudException; for sql (track2) --> ManagementException.
It will be only exist the MangementException after we migrate mysql into track2.